<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionWebcam"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="separator"/>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionRewind"/>
    <addaction name="separator"/>
    <addaction name="actionForward"/>
   </widget>
   <widget class="QMenu" name="menuSound">
    <property name="title">
     <string>Sound</string>
    </property>
    <addaction name="actionMute"/>
    <addaction name="separator"/>
    <addaction name="actionUnmute"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionsettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVideo"/>
   <addaction name="menuSound"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionWebcam"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="separator"/>
   <addaction name="actionPause"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionMute"/>
   <addaction name="separator"/>
   <addaction name="actionUnmute"/>
   <addaction name="separator"/>
   <addaction name="actionRewind"/>
   <addaction name="separator"/>
   <addaction name="actionForward"/>
   <addaction name="separator"/>
   <addaction name="actionsettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/mute.png</normaloff>:/icons/icons/mute.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionUnmute">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/unmute.png</normaloff>:/icons/icons/unmute.png</iconset>
   </property>
   <property name="text">
    <string>Unmute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRewind">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/rewind.png</normaloff>:/icons/icons/rewind.png</iconset>
   </property>
   <property name="text">
    <string>Rewind</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/forward.png</normaloff>:/icons/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionWebcam">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/webcam.png</normaloff>:/icons/icons/webcam.png</iconset>
   </property>
   <property name="text">
    <string>Webcam</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionsettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/help.png</normaloff>:/icons/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
